translations/Marathi/special-terms.md

```diff
@@ -0,0 +1,42 @@
+﻿Terms
+
+Message : संदेश , Most appropriate synonym for general usage in zulip, <username> ला एक संदेश पाठवा
+Private Message (PM) : खाजगी संदेश , Best translation of private to imply it's meaning , <username> ला खासगी एक संदेश पाठवा
+Group PM : ग्रुप खाजगी संदेश  , Transliteration used for group , <group name> मध्ये एक ग्रुप खाजगी संदेश पाठवा
+Realm : संघटन , used translation of organization for same reason as of chinese translation , एक संघटन बनवा
+Stream : स्ट्रीम  , used transliteration for better understanding , स्ट्रीममध्ये प्रवेश करा
+Topic : विषय , best word to signify the usage in zulip , नवीन विषय सुरू करा
+Private/Invite-Only Stream : खाजगी / केवळ-आमंत्रित स्ट्रीम , used translation of "private/invited-only" for better understanding , आपले खाजगी / केवळ-आमंत्रित स्ट्रीम
+Public Stream : सार्वजनिक स्ट्रीम  , best translation of public for usage in zulip , आपले सार्वजनिक स्ट्रीम 
+Bot : बॉट , used transliteration , स्वागत बॉट
+Integration : अनुप्रयोग एकीकरण , used translation of "application integration" , फेसबुक सह झुलीप अनुप्रयोग एकात्मता
+Notification : सूचना , translates to information but is understandable by user , सूचना सक्रिय करा
+Alert Word : सतर्क शब्द , only translation which is understandable by average user , आपले सतर्क शब्द
+View : पहा , used translation of see for implication that it is verb , संदेश पहा
+Home : मुख्यपृष्ठ , used translation of homepage instead for better understanding , मुख्यपृष्ठावर जा
+Emoji : इमोजी , used transliteration as there is no translation of this word , इमोजी शैली
+
+Phrases
+
+Subscribe to a stream/Unsubscribe from a stream : एखाद्या स्ट्रीमची सदस्यता घ्या / एक स्ट्रीम मधुन सदस्यता रद्द करा , only translation available , <stream name> हया स्ट्रीमची सदस्यता घ्या 
+Narrow to : याद्वारे फिल्टर करा , used translation of filter by , याद्वारे फिल्टर करा
+Filter : फिल्टर , used transliteration , याद्वारे फिल्टर करा
+Mute/Unmute : सूचना चालू / बंद करा , used translation of "turn on/turn off notifications", सूचना चालू / बंद करा
+Deactivate/Reactivate : बंद करा / सुरू करा  ,  used translation of "Turn OFF / ON", आपले खाते बंद करा / सुरू करा
+Search : शोध , same as translation used in google search , वापरकर्त्यासाठी शोधा
+Pin : पिन करा , the second word force-implies the verb form , आपले पोस्ट पिन करा
```

Comment:
Here it should be 'आपली' instead of 'आपले'

Reply:
Actually both are accepted but it would be more appropriate to give the word 'post' masculine gender.
PS for non speakers: Marathi and most other Indian languages do not have provisions for neutral or common genders that is used in English. Therefore inanimate objects are always assigned a gender (only male/female)